community/zathura-pdf-poppler/depends

```diff
@@ -0,0 +1,4 @@
+meson make
+ninja make
```

Comment:
This line can now be removed due to my changes here: https://github.com/kisslinux/repo/commit/e27b638dda62dd9f090c978507d01fa5237e5a45